### DISCUSSION TITLE
Freetz-ng on Fritz!Box 4020

### DISCUSSION THREAD

**Discussion by GSE-404** [Freetz-NG/freetz-ng#615] [Allgemein]
Hi, I have a 4020 International Edition Fritz!box that I want to customize with Freetz-ng.

I found the 4020 box among the supported devices, but apparently only the German variant of the box is supported by Freetz-ng:
https://freetz-ng.github.io/freetz-ng/FIRMWARES.html#fritzbox-wan-4xxx
Fritz!Box 4020
147.06.83 rev43440 {GER}
147.07.02 rev93867 {GER}

I noticed that the latest German and international AVM firmwares (07.02) are the same (checked with MD5).
So I decided to generate a basic 07.02 image and tried to flash it via the fritz!box web Gui. 
Unfortunately, I have always this error messaget:
"The specified file does not contain a Fritz!OS released for this device by AVM."

Freetz version I use:
freetz-ng 20218-327247314 master 2022-09-16

Current firmware version of my 4020 box:
Fritz!OS 07.02

Freetz make output:
STEP 3: PACK/SIGN
  checking for left over version-control-system files
  integrate freetz info file into image
packing var.tar
  checking signature key files
    adding public signature key file
creating filesystem image (SquashFS4-xz)
  SquashFS block size: 64 kB (65536 bytes)
merging kernel image
  kernel image size: 14.1 MB, max 14.9 MB, free 0.8 MB (823552 bytes)
adding checksum to kernel.image
packing images/4020_07.02.ger_freetz-ng-20218-327247314_20220919-130417.image
  packed image file size: 16.1 MB (16885760 bytes)
signing packed .image file
  signed image file size: 16.1 MB (16885760 bytes)
source firmware: 4020_de-es-fr-it-pl 147.07.02 rev93867 {GER} (10.01.2022 11:28:04)
  source image file size: 16.8 MB (17582080 bytes)
done.
FINISHED

Is there something specific I've missed to do?

**Comment by fda77** [comment]
Just googel for "The specified file does not contain a Fritz!OS released for this device by AVM."
